Hmm.. `check-types` is failing on `src/lib/api.js` with: 
```
Error: src/lib/api.ts(94,3): error TS2322: Type '(path: string, options?: FetchOptions) => Promise<Response>' is not assignable to type '(input: RequestInfo | URL, init?: RequestInit | undefined) => Promise<Response>'.
```
But I didn't touch this file 🤔

--- a/src/lib/client-file-uploader.ts
+++ b/src/lib/client-file-uploader.ts
@@ -28,9 +28,9 @@ const fetchWithRetry: ( input: RequestInfo | URL, init?: RequestInit ) => Promis
 	// eslint-disable-next-line @typescript-eslint/no-unsafe-call
 	require( 'fetch-retry' )( fetch, {
 		// Set default retry options
-		retries: 5,
+		retries: 3,
 		retryDelay: ( attempt: number, error: Error, response: Response ) => {
-			return Math.pow( 2, attempt ) * 500; // 500, 1000, 2000
+			return Math.pow( 2, attempt ) * 1000; // 1000, 2000, 4000
 		},
 	} );
 

--- a/src/lib/client-file-uploader.ts
+++ b/src/lib/client-file-uploader.ts
@@ -29,7 +29,9 @@ const fetchWithRetry: ( input: RequestInfo | URL, init?: RequestInit ) => Promis
 	require( 'fetch-retry' )( fetch, {
 		// Set default retry options
 		retries: 5,
-		retryDelay: 1000,
+		retryDelay: ( attempt: number, error: Error, response: Response ) => {
+			return Math.pow( 2, attempt ) * 500; // 500, 1000, 2000
+		},
 	} );
 
 const debug = debugLib( 'vip:lib/client-file-uploader' );

--- a/src/lib/client-file-uploader.ts
+++ b/src/lib/client-file-uploader.ts
@@ -7,7 +7,7 @@ import { constants, createReadStream, createWriteStream, type ReadStream } from 
 import { access, mkdtemp, open, stat } from 'node:fs/promises';
 import os from 'os';
 import path from 'path';
-import fetch, { HeaderInit, RequestInit, Response } from 'node-fetch';
+import fetch, { HeaderInit, RequestInfo, RequestInit, Response } from 'node-fetch';
 import chalk from 'chalk';
 import { createGunzip, createGzip } from 'zlib';
 import { createHash } from 'crypto';
@@ -15,7 +15,6 @@ import { pipeline } from 'node:stream/promises';
 import { PassThrough } from 'stream';
 import { Parser as XmlParser } from 'xml2js';
 import debugLib from 'debug';
-import fetchRetry from 'fetch-retry';
 
 /**
  * Internal dependencies

